cypress/integration/LeftNavigationClickEvent.feature

```diff
@@ -19,12 +20,15 @@ Feature: Left Navigation click events
             | prop4    | D=pev1              |
             | evar2    | english             |
             | prop8    | english             |
-            | channel  | About Cancer        |
-        And pageName and pageURL parameters contain current url
+          
+        # And the following parameters contain current url
```

Comment:
Can you please remove these if they are no longer needed? The same in the other files.